tests/test_data_utils/test_load_csv_file.py

```diff
@@ -44,13 +44,12 @@ def tearDownClass(cls):
 
     def test_load_single_csv_file(self):
         result = load_csv_files(self.valid_file)
-        self.assertIsInstance(result, list)
-        self.assertEqual(len(result), 1)
-        self.assertIsInstance(result[0][1], pd.DataFrame)
+        self.assertIsInstance(result, pd.DataFrame)
 
     def test_load_multiple_csv_files(self):
         result = load_csv_files([self.valid_file, self.valid_file])
-        self.assertEqual(len(result), 2)
+        self.assertIsInstance(result, pd.DataFrame)
+        self.assertEqual(len(result), 4)

```

Comment:
@ruiheesi, wouldy you pleasse assert that the column "loaded file name" exists in the returned dataframe and in case you loaded two files, you have two unique values and they are correct?

Reply:
@ruiheesi , I am am not sure why the correct value is 4?

Reply:
@ruiheesi, would you pass two files with different filenames so that we can test that we are appending the correct extra column "loaded file name" with appropriate values?

Reply:
@georgezakinih Hi George, each dataset has two rows, hence combined will have 4 rows.